### ISSUE TITLE
Only the active page has border on Safari

### ISSUE THREAD

**Issue by seopei** [stephanrauh/ngx-extended-pdf-viewer#2657]
On Safari (Mac and iOS) only the active page has a visible border. Reproduce: https://pdfviewer.net/extended-pdf-viewer/simple

https://github.com/user-attachments/assets/fecadd2e-47d4-4ca3-bd5e-36368bb2538e

**Comment by avdoseferovic** [comment]
I fixed this by adding a border-bottom property

```
::ng-deep .pdfViewer .page {
  border-bottom: 9px solid #f5f5f5 !important;
}
```

**Comment by stephanrauh** [comment]
What a strange bug!

**Comment by stephanrauh** [comment]
Interesting: I've asked ChatGPT, and it pointed me to another issue in this repository. It seems other developers ran into the same problem and found a solution. I'll add that solution to version 22.

**Comment by stephanrauh** [comment]
The bugfix has landed with version 22.0.1.
